website/src/content/api-reference/custom-webpack-config.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Custom Webpack Config"
+---
+
+<Subtitle>
+The utilities allows you create a custom webpack configuration for a Merchant Center Application.
+</Subtitle> 
+
+# Installation
+
+```
+yarn add @commercetools-frontend/mc-scripts
+
+# or
+
+npm --save install @commercetools-frontend/mc-scripts
+```
```

Comment:
This part is not really necessary as `mc-scripts` is anyway required to be installed. I would focus more on describing the need and use cases for customizing the webpack config (which I'm missing a bit in this document).

For example:

```suggestion
By default Custom Applications include a webpack configuration to handle most of the common features like code transformation via Babel, loading CSS modules, loading images and SVGs, loading GraphQL documents, etc.

However, it could be that your application requires some extra functionality like loading a particular file type, using a specific webpack plugin or simply include other source files in your repository.

In order to extend the functionalities of webpack, you can define specific files in the root of your Custom Application project.
```